### PR TITLE
Lr2

### PR DESCRIPTION
**Pull request by Daniil1605**

### CODE REVIEW

File: Lab 02/proj.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="utf-8">
+  <title>Пример страницы</title>
+  <link rel="stylesheet" href="proj.css" >
```
Comment (Daniil1605) [inline Lab 02/proj.html:7]: Need one line to make responsivness

File: Lab 02/proj.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="utf-8">
+  <title>Пример страницы</title>
+  <link rel="stylesheet" href="proj.css" >
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+ </head>
+ <body>
+   <div class="head-menu-container-computer">
```
Comment (Daniil1605) [inline Lab 02/proj.html]: https://www.w3schools.com/html/html5_semantic_elements.asp

File: Lab 02/proj.css [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+}
+
+.Blazrobar-link-group {
+	width: 100%;
+	height: auto;
+	overflow: auto;
+	background:#078ba4;
+}
+
+.Blazrobar-link-group h1 {
```
Comment (Daniil1605) [inline Lab 02/proj.css:70]: one class naming style

File: Lab 02/proj.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  <meta charset="utf-8">
+  <title>Пример страницы</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="proj.css" >
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+ </head>
+ <body>
+   <header>
+   <div class="Head-menu-container-computer">
+   <div class="Head-menu-block-computer" >PSD WEBSITES </div>
```
Comment (Daniil1605) [inline Lab 02/proj.html]: ul > li > a PSD WEbb...

**Comment by aliaksandrkrasnou** [comment]
Approved, okay 👍

**Pull request closed without merging**